src/lib.rs

```diff
@@ -127,6 +127,20 @@ impl Alpm {
     })
   }
 
+  /// Compare two version strings and determine which one is newer.
+  ///
+  /// Returns a value comparable to the way strcmp works. Returns 1
+  /// if a is newer than b, 0 if a and b are the same version, or -1
+  /// if b is newer than a.
+  pub fn vercmp(&self, a: String, b: String) -> std::io::Result<i32> {
+    unsafe {
+      // int alpm_pkg_vercmp(const char *a, const char *b)
+      let pkg_vercmp: Symbol<fn(*const c_char, *const c_char) -> *const c_int> = try!( self.lib.get(b"alpm_pkg_vercmp") );
+
+      Ok(pkg_vercmp(a.as_ptr() as *const c_char, b.as_ptr() as *const c_char) as i32)
```

Comment:
Maybe we should wrap the result in [std::cmp::Ordering](https://doc.rust-lang.org/stable/std/cmp/enum.Ordering.html).

What do you think?